Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```diff
@@ -200,6 +200,7 @@
     tags:
     - ClownMask
     - WhitelistChameleon
+    - IPCMaskWearable # Estacao Pirata - IPCs
```

Comment:
adding a tag like this will be a nightmare to maintain and very prone to breaking when things get updated, why is this done?

Reply:
IPCs cannot wear a majority of masks. The system actually whitelists IdentityBlocker clothing and welding masks, so this shouldn't actually be here on a majority of prototypes. I'll just remove the whitelist since I can't see any particular reason for it to exist